codegen/lib/coin_test_gen.rb

```diff
@@ -14,8 +14,24 @@ def initialize()
   # Transforms a coin name to a C++ name
   def format_name(n)
     formatted = n
-    #formatted = formatted.sub(/^([a-z]+)/, &:upcase)
-    formatted = formatted.sub(/\s/, '')
+
+    # Remove whitespaces
+    formatted.gsub!(/\s+/, '')
+
+    exceptions = {
+      "AvalancheC-Chain" => "Avalanche",
+      "CronosChain" => "Cronos",
+      "ECOChain" => "ECO",
+      "KuCoinCommunity" => "KuCoin",
+      "SmartBitcoinCash" => "Bitcoin",
+      "SmartChain" => "Binance",
+      "SmartChainLegacy" => "Binance",
+    }
```

Comment:
We should not have such exceptions in the script. Maybe use displayName field? Or introduce a new optional folderName field?

Reply:
I agree, maybe we can rename these folders as well, only coin type tests affected, is that correct?